packages/scandipwa/src/route/UrlRewrites/UrlRewrites.container.js

```diff
@@ -78,11 +78,11 @@ export class UrlRewritesContainer extends PureComponent {
 
     __construct(props) {
         super.__construct(props);
```

Comment:
@zans-laksa @leonazava 

Is it necessary to keep a constructor which only calls the parent constructor, and nothing else?

Reply:
No, it's useless.